lib/src/main/java/com/hootsuite/nachos/NachoTextView.java

```diff
@@ -512,7 +516,8 @@ public boolean onTouchEvent(@NonNull MotionEvent event) {
         int action = event.getAction();
         clearChipStates();
         Chip touchedChip = findTouchedChip(event);
-        if (touchedChip != null && isFocused() && action == MotionEvent.ACTION_UP) {
+        boolean singleTapUp = gestureDetector.onTouchEvent(event);
```

Comment:
no need to create a singleTapUp variable, you can write && gestureDetector.onTouchEvent(event) directly.